src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -1310,13 +1308,17 @@ void InsertConcatAligningFilterPass::run() {
                 }
 
                 auto num_rows_in = dims[1];
-                size_t aligned64_offset = std::max(0, static_cast<int>(ALIGN64(offset) - 64));
-                size_t num_rows_padded = (offset - aligned64_offset) / bytesPerConcatElement;
+                size_t aligned_offset =
```

Comment:
I am affraid that if you will not add `static_cast<size_t>(std::max)` will introduce new warning